src/Pos/Client/Txp/History.hs

```diff
@@ -237,15 +236,16 @@ getBlockHistoryDefault
     :: forall ssc ctx m. TxHistoryEnv' ssc ctx m
     => [Address] -> m (DList TxHistoryEntry)
 getBlockHistoryDefault addrs = do
-    bot <- GS.getBot
-    sd <- GS.getSlottingData
-    systemStart <- getSystemStart
+
+    systemStart <- getSystemStartM
+    bot         <- GS.getBot
+    sd          <- GS.getSlottingData
```

Comment:
I think @gromakovsky wrote that we have in in both places since we need to pull it into memory at some point if the node get's shut down. Don't really know where that place is :grin: